packages/components/src/Chip/Chip.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import styles from "./Chip.css";
+import { Avatar, AvatarProps } from "../Avatar";
+import { Text } from "../Text";
+import { ButtonDismiss } from "../ButtonDismiss";
+
+interface DismissibleAction {
+  ariaLabel: string;
+  onDismiss(): void;
+}
+
+interface ChipProps {
+  label: string;
+  avatar?: Exclude<AvatarProps, "size">;
+  dismissAction: DismissibleAction;
```

Comment:
Had a little sudden realization here. All of your tests and examples mentions the aria label as "Remove"

![image](https://user-images.githubusercontent.com/15986172/131540404-b2e94d96-832d-4ef7-acde-50c49cdad1af.png)
![image](https://user-images.githubusercontent.com/15986172/131540495-062c9932-62aa-4e9c-ab2c-29a46dd0651c.png)

Should this prop be called remove instead of dismiss?

Reply:
The `ariaLabel` specified in the tests haven't been selected explicitly. However I do believe the prop name could be changed, I like using `onDismiss` because it is consistent with the `ButtonDismiss` component, however always welcome to change if a different name is better

Reply:
Ah! Also realized that `ButtonDismiss` is a private component. In that case, I'd follow the components that already uses it. Which is the popover and modal that both calls `onRequestClose` so something along the lines of `onRequest...`. `onRequestDismiss`?

Anyway, my point about your example aria label is going with the more natural word that someone consuming this would use. The fact that you used "Remove" instead of "Dismiss" as an example of how would someone use this kinda suggests that this should probably be "remove". I don't have a strong opinion on it tho, just thinking about the predictability of it when the non-atlantis friends are consuming it since they wouldn't know that `ButtonDismiss` exists.